src/sentry/rules/processing/delayed_processing.py

```diff
@@ -269,7 +270,10 @@ def apply_delayed(project_id: int) -> None:
 
     # STEP 3: Fetch the Rule models we need to check
     alert_rules = Rule.objects.filter(id__in=list(rules_to_groups.keys()))
-
+    snoozed_rules = RuleSnooze.objects.filter(rule__in=alert_rules, user_id=None).values_list(
+        "rule", flat=True
+    )
```

Comment:
Nit: Let's convert this to a set?

Reply:
I'm about to do follow ups in this file so I'll change it there!